### ISSUE TITLE
'api.crowdin.com: System error' when trying to download translations

### ISSUE THREAD

**Issue by dshiryaev-plesk** [crowdin/crowdin-cli#430]
**Describe the bug**
Hello,
The CLI periodically fails with the error when we try to download translations for different our projects.

**To Reproduce**
1. Configuration file: 
```
api_token_env: CROWDIN_PERSONAL_TOKEN
base_path: /opt/tailor/psdk/ext-acronis-backup/wd/po
files:
- ignore:
  - /[a-z]{2}-[A-Z]{2}/**/*.po
  import_eq_suggestions: 1
  languages_mapping:
    locale_with_underscore:
      ar: ar_AE
      fil: tl_PH
  source: /ext-acronis-backup/**/*.po
  translation: /%locale_with_underscore%/%original_path%/%original_file_name%
preserve_hierarchy: true
project_id: 433620
```

2. Execute CLI command: `crowdin download --config crowdin.yml`
3. See an error
```
2022-02-04T08:31:58.368711792+07:00 stdout F Running 'crowdin download --config crowdin.yml'...
2022-02-04T08:32:00.171406582+07:00 stdout F [?????] Fetching project info                             [?????] Fetching project info                             [?????] Fetching project info                             ? Fetching project info
2022-02-04T08:32:00.172052422+07:00 stderr F ? Failed to collect project info. Please contact our support team for help
2022-02-04T08:32:00.172063234+07:00 stderr F ? Error from server: <Code: <empty_code>, Message: api.crowdin.com: System error>
```

**Expected behavior**
Translations has been downloaded without errors.

**Environment:**
 - OS: alpine
 - Version: 3.9
 - Base Docker image: 
     - https://hub.docker.com/r/melopt/alpine-perl-devel
     - https://github.com/melo/docker-alpine-perl-runtime/blob/master/Dockerfile
 - Kubernetes

**Additional context**
Version of Crowdin CLI: 3.7.2

**Comment by andrii-bodnar** [comment]
Hi @dshiryaev-plesk, could you please run the CLI command with the `--debug` parameter? It should provide more details about the error.

**Comment by dshiryaev-plesk** [comment]
@andrii-bodnar is there a way to enable the debug mode using an environment variable or an option in the config file ?

**Comment by andrii-bodnar** [comment]
@dshiryaev-plesk, unfortunately, no. Only as a command parameter.

**Comment by dshiryaev-plesk** [comment]
@andrii-bodnar, where should I set the --debug option in the command, after `crowdin` or after `download` ? Currently I don't see any effect in the outputs of successfull runs.

**Comment by andrii-bodnar** [comment]
@dshiryaev-plesk like this:

```
crowdin download --config crowdin.yml --debug
```

It should show more data in the unsuccessful runs.

**Comment by dshiryaev-plesk** [comment]
Hello @andrii-bodnar,
Here is the error log for running the CLI with the `--debug` option:
```
2022-02-18T13:11:12.462024003+07:00 stdout F Running 'crowdin --debug download --config crowdin.yml'...	
2022-02-18T13:11:14.125844642+07:00 stdout F [?????] Fetching project info                             [?????] Fetching project info                             [?????] Fetching project info                             ? Fetching project info	
2022-02-18T13:11:14.127112169+07:00 stderr F java.lang.RuntimeException: Failed to collect project info. Please contact our support team for help	
2022-02-18T13:11:14.127290124+07:00 stderr F 	at com.crowdin.cli.utils.console.ConsoleSpinner.execute(ConsoleSpinner.java:29)	
2022-02-18T13:11:14.127323342+07:00 stderr F 	at com.crowdin.cli.commands.actions.DownloadAction.act(DownloadAction.java:93)	
2022-02-18T13:11:14.127346359+07:00 stderr F 	at com.crowdin.cli.commands.actions.DownloadAction.act(DownloadAction.java:58)	
2022-02-18T13:11:14.127393585+07:00 stderr F 	at com.crowdin.cli.commands.picocli.GenericActCommand.run(GenericActCommand.java:37)	
2022-02-18T13:11:14.127424711+07:00 stderr F 	at picocli.CommandLine.executeUserObject(CommandLine.java:1939)	
2022-02-18T13:11:14.127473661+07:00 stderr F 	at picocli.CommandLine.access$1300(CommandLine.java:145)	
2022-02-18T13:11:14.127497202+07:00 stderr F 	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)	
2022-02-18T13:11:14.127571758+07:00 stderr F 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)	
2022-02-18T13:11:14.127588831+07:00 stderr F 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2311)	
2022-02-18T13:11:14.12762487+07:00 stderr F 	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)	
2022-02-18T13:11:14.127658416+07:00 stderr F 	at picocli.CommandLine.execute(CommandLine.java:2078)	
2022-02-18T13:11:14.127673987+07:00 stderr F 	at com.crowdin.cli.commands.picocli.PicocliRunner.execute(PicocliRunner.java:37)	
2022-02-18T13:11:14.127717988+07:00 stderr F 	at com.crowdin.cli.Cli.main(Cli.java:16)	
2022-02-18T13:11:14.12787657+07:00 stderr F Caused by: java.lang.RuntimeException: Error from server: <Code: <empty_code>, Message: api.crowdin.com: System error>	
2022-02-18T13:11:14.127883285+07:00 stderr F 	at com.crowdin.cli.client.CrowdinClientCore.executeRequest(CrowdinClientCore.java:125)	
2022-02-18T13:11:14.127904105+07:00 stderr F 	at com.crowdin.cli.client.CrowdinClientCore.executeRequest(CrowdinClientCore.java:100)	
2022-02-18T13:11:14.127954752+07:00 stderr F 	at com.crowdin.cli.client.CrowdinProjectClient.getProject(CrowdinProjectClient.java:96)	
2022-02-18T13:11:14.127963856+07:00 stderr F 	at com.crowdin.cli.client.CrowdinProjectClient.populateProjectWithInfo(CrowdinProjectClient.java:80)	
2022-02-18T13:11:14.128006874+07:00 stderr F 	at com.crowdin.cli.client.CrowdinProjectClient.downloadFullProject(CrowdinProjectClient.java:45)	
2022-02-18T13:11:14.128019082+07:00 stderr F 	at com.crowdin.cli.utils.console.ConsoleSpinner.execute(ConsoleSpinner.java:24)	
2022-02-18T13:11:14.128027112+07:00 stderr F 	... 12 more	
2022-02-18T13:11:14.998314734+07:00 stdout F 	
2022-02-18T13:11:14.998373939+07:00 stdout F ????????????????????????????????????????????????????????????????????????????????	
2022-02-18T13:11:14.998382132+07:00 stdout F ?           New version of Crowdin CLI is available! 3.7.2 -> 3.7.7            ?	
2022-02-18T13:11:14.99840296+07:00 stdout F ????????????????????????????????????????????????????????????????????????????????	
2022-02-18T13:11:14.99840756+07:00 stdout F ?      Changelog: https://github.com/crowdin/crowdin-cli/releases/latest       ?	
2022-02-18T13:11:14.998411468+07:00 stdout F ?                    Please update for the best experience!                    ?	
2022-02-18T13:11:14.998415769+07:00 stdout F ????????????????????????????????????????????????????????????????????????????????
```
